src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/Message.vue

```diff
@@ -56,3 +71,10 @@ export default defineComponent({
   },
 });
 </script>
+<style>
```

Comment:
Do we need this or can we replace `<hr class="grey-line" />` with `<v-divider />` you can set color, etc. if you need to:

https://vuetifyjs.com/en/api/v-divider/#links

Reply:
was already using  <v-divider> for "mdAndUp", forgot to update it for "smAndDown", fixed now, removed unused styling as well